requirements/_docs.txt

```diff
@@ -29,5 +29,4 @@ pydantic > 1.0.0, < 3.0.0
 # --------------------------
 # todo: until this has resolution - https://github.com/sphinx-gallery/sphinx-gallery/issues/1290
 # Image
-scikit-image ~=0.22; python_version < "3.10"
-scikit-image ~=0.25; python_version > "3.9"  # we do not use `> =` because of oldest replcement
+scikit-image ==0.25.*
```

Comment:
```suggestion
scikit-image ~=0.22; python_version < "3.10"
scikit-image ~=0.25; python_version > "3.9"  # we do not use `> =` because of oldest replcement
```